examples/ensemble/plot_gradient_boosting_cv.py

```diff
@@ -0,0 +1,97 @@
+"""
+===================================
+Early stopping of Gradient Boosting
+===================================
+
+Gradient boosting works by fitting a regression tree at each iteration to
+improve its prediction over the training data. The improvement in prediction
+succesively reduces as more regression trees are added. The
+:class:`~sklearn.ensemble.GradientBoostingClassifierCV` class halts the
+training when there is no significant improvement in accuracy on the validation
+data during the last few iterations. This example illustrates how the 
+GradientBoostingClassifierCV class can acheive almost the same accuracy as
+compared to the GradientBoostingClassifier class with significantly lesser
```

Comment:
the phrasing is odd here. Also, it should achieve the same accuracy, shouldn't it?